### PR TITLE
docs: add config-consul doc

### PR DESCRIPTION
**Pull request by ahaostudy**
#### What type of PR is this?

docs

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
+---
+title: "Consul"
+date: 2024-03-02
+weight: 1
```
Comment (li-jin-gou) [inline content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: 调整一下 weight

Comment (ahaostudy) [reply content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: 这个有什么顺序要求吗

Comment (ahaostudy) [reply content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: 之前的 weight 漏了一个 1，我就直接用 1 了

**Comment by li-jin-gou** [comment]
放到最后，调整一下顺序



---Original---
From: ***@***.***&gt;
Date: Sun, Mar 3, 2024 19:04 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;"Review ***@***.***&gt;;
Subject: Re: [cloudwego/cloudwego.github.io] docs: add config-consul doc (PR#993)




 
@ahaostudy commented on this pull request.
 
 
In content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:
 &gt; @@ -0,0 +1,421 @@ +--- +title: "Consul" +date: 2024-03-02 +weight: 1  
这个有什么顺序要求吗

File: content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md (hunk 1 above)
Comment (li-jin-gou) [reply content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: 放到最后，调整一下顺序

Comment (li-jin-gou) [reply content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: weight 1 是这个问题件 https://github.com/cloudwego/cloudwego.github.io/blob/main/content/zh/docs/kitex/Tutorials/service-governance/config-center/_index.md 并没有漏

Comment (ahaostudy) [reply content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: > 放到最后，调整一下顺序

OK

Comment (ahaostudy) [reply content/zh/docs/kitex/Tutorials/service-governance/config-center/consul.md:4]: > weight 1 是这个文件 https://github.com/cloudwego/cloudwego.github.io/blob/main/content/zh/docs/kitex/Tutorials/service-governance/config-center/_index.md 并没有漏

_index weight 配置的应该是在当前目录在父级上的 weight ？

File: content/en/docs/kitex/Tutorials/service-governance/config-center/_index.md [hunk 2]
```diff
@@ -18,13 +18,14 @@ Microservice developers can use the configuration center to dynamically obtain s
...
-|    apollo     | [config-apollo](https://github.com/kitex-contrib/config-apollo) |
-|     file      | [config-file](https://github.com/kitex-contrib/config-file)  |
+| config-center |                              depository                              |
+|:-------------:|:--------------------------------------------------------------------:|
+|     nacos     |    [config-nacos](https://github.com/kitex-contrib/config-nacos)     |
+|     etcd      |     [config-etcd](https://github.com/kitex-contrib/config-etcd)      |
+|    apollo     |   [config-apollo](https://github.com/kitex-contrib/config-apollo)    |
+|     file      |     [config-file](https://github.com/kitex-contrib/config-file)      |
 |   zookeeper   | [config-zookeepr](https://github.com/kitex-contrib/config-zookeeper) |
+|    consul     |  [config-zookeepr](https://github.com/kitex-contrib/config-consul)   |
```
Comment (AsterDY) [inline content/en/docs/kitex/Tutorials/service-governance/config-center/_index.md:28]: image
consul的第二列不对吧

File: content/zh/docs/kitex/Tutorials/service-governance/config-center/_index.md [hunk 3]
```diff
@@ -18,13 +18,14 @@ kitex-contrib 提供了对于社区主流配置中心的拓展，实现了动态
...
-|   file    |  [config-file](https://github.com/kitex-contrib/config-file)     |
-| zookeeper |  [config-zookeepr](https://github.com/kitex-contrib/config-zookeeper) |
+|   配置中心    |                                  仓库                                  |
+|:---------:|:--------------------------------------------------------------------:|
+|   nacos   |    [config-nacos](https://github.com/kitex-contrib/config-nacos)     |
+|   etcd    |     [config-etcd](https://github.com/kitex-contrib/config-etcd)      |
+|  apollo   |   [config-apollo](https://github.com/kitex-contrib/config-apollo)    |
+|   file    |     [config-file](https://github.com/kitex-contrib/config-file)      |
+| zookeeper | [config-zookeepr](https://github.com/kitex-contrib/config-zookeeper) |
+|  consul   |  [config-zookeepr](https://github.com/kitex-contrib/config-consul)   |
```
Comment (AsterDY) [inline content/zh/docs/kitex/Tutorials/service-governance/config-center/_index.md:28]: 同上

File: content/en/docs/kitex/Tutorials/service-governance/config-center/_index.md (hunk 2 above)
Comment (ahaostudy) [reply content/en/docs/kitex/Tutorials/service-governance/config-center/_index.md]: 修了，之前没注意哈哈

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**